WordPress/Classes/ViewRelated/Post/PostListFilterSettings.swift

```diff
@@ -0,0 +1,166 @@
+import Foundation
+import WordPressComAnalytics
+
+@objc class PostListFilterSettings: NSObject {
```

Comment:
`@objc` is only necessary if your class does not inherit from `NSObject`.  We should be able to safely remove the `@objc` here.